ci/citest/acceptance-test/multibox/ind-steps/step-kvm/common.source

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# possibly useless to wrap this
+function kvm_enbabled {
+    lsmod | grep kmv*
+    return $?
+}
+
+function nested_enabled {
+    local disabled=false
+
+    case ${ACCEL_TYPE} in
+        vmx) [[ "$(cat /sys/module/kvm/intel/parameters/nested)" == "Y" ]] || disabled=true ;;
+        svm) [[ $(cat /sys/module/kvm_amd/parameters/nested) -eq 1 ]] || disabled=true ;;
+    esac
+
+    return $disabled
+}
```

Comment:
At first I was not sure wether the scripts should try to enable kvm/the nested parameter or not, opinions on this? This also ties into the comment about kvm/qemu being separate steps.
I enabled nested manually and I'm not even sure it can be done from within the docker container